### PR TITLE
Replace old account page with new one - Closes #1780

### LINKED ISSUE #1780: Replace old account page with new one
## Specification of the new account page
- The account page should be similar to "My Wallet" page
- There should be always "Wallet" tab - same as in "My Wallet"
https://netguru.invisionapp.com/share/7MQH1EPBPK8#/screens/346840650
- There should be always "Votes" tab that lists all delegates that the account voted for, ordered by rank from smaller to bigger. "Show more" should work like in transactions list - show 30 at a time.
https://netguru.invisionapp.com/share/NRQVO14USKQ#/screens
- "Filter by name" in "Votes" tab should show delegates that contain the entered string in the delegate name.
- There should be "Delegate" tab with delegate stats if the account is a delegate
https://netguru.invisionapp.com/share/GDQUS2ST9BH#/screens
- The "Follow" button should work as described in https://github.com/LiskHQ/lisk-hub/issues/1805
- "Send LSK to this Wallet" opens the send page with address pre-filled.

### PR DESCRIPTION
**Pull request by massao**
### What issue have I solved?

#1780

### How have I implemented/fixed it?

Replaced old account page with new one, and updated wallet to have votes tab, and delegate tab when needed.

### How has this been tested?

Search any account in the search bar, and should open on the new layout.
Search delegate account, and should show the delegate tab on the new layout.
Wallet page should have Votes tab.
Login with a delegate account, and the delegate tab should appear on the wallet page.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

### CODE REVIEW

File: test/constants/selectors.js [hunk 1]
```diff
@@ -24,7 +24,8 @@ const ss = {
   delegateStatisticsTab: '.delegate-statistics ',
   votedAddress: '.votes .voter-address',
   voterAddress: '.voters .voter-address',
-  accountAddress: '.copy-title',
+  voteRow: '.votes-row',
```
Comment (osvaldovega) [inline test/constants/selectors.js]: I think there is a typo in here, is **vote** or **votes**

File: test/cypress/e2e/account.spec.js [hunk 2]
```diff
@@ -18,41 +18,39 @@ describe('Account', () => {
     // https://github.com/cypress-io/cypress/issues/695
     cy.wait(1000);
     cy.url().should('contain', '16313739661666666666L');
-    cy.get(ss.accountAddress).eq(1).contains('16313739661666666666L');
-    cy.get(ss.accountBalance).eq(1).contains('0');
+    cy.get(ss.accountAddress).contains('16313739661666666666L');
```
Comment (osvaldovega) [inline test/cypress/e2e/account.spec.js]: in line 35 you are using constants but not in here, maybe you can use constants in here too

File: test/cypress/e2e/activity.spec.js [hunk 3]
```diff
@@ -59,6 +60,7 @@ function testActivity(open) {
   describe('Account info tab', () => {
     beforeEach(() => {
       cy.autologin(accounts.genesis.passphrase, networks.devnet.node);
+      cy.visit('/dashboard');
```
Comment (osvaldovega) [inline test/cypress/e2e/activity.spec.js]: here you can use constants too

File: test/cypress/e2e/activity.spec.js [hunk 4]
```diff
@@ -150,31 +118,15 @@ function testDelegateActivity(open) {
...
-      cy.get(ss.voterAddress).eq(1).click();
-      cy.get(ss.leftBlockAccountExplorer).find(ss.accountAddress).should('have.text', '16313739661670634666L');
-    });
-
     it('Shows statistics', () => {
       cy.get(ss.delegateStatsUptime).contains('100%');
       cy.get(ss.delegateStatsUptime).contains('1');
       cy.get(ss.delegateStatsApproval).contains('100%');
-      cy.get(ss.delegateStatsWeight).contains('100,000,000');
+      // cy.get(ss.delegateStatsWeight).contains('100,000,000');
```
Comment (osvaldovega) [inline test/cypress/e2e/activity.spec.js]: do you need this line or can be remove it?

**Comment by Efefefef** [comment]
@slaweet No specification

**Comment by Efefefef** [comment]
@slaweet 
1)Why wallet not account?
image
2) Last forged block - date or countdown?
3) Where shoul I be redirected after clicking Send LSK to this Wallet (again why Wallet?) and completing the transaction? My wallet or his wallet?
🐛 It is said that "On your dashboard" checkbox should not be present now
https://github.com/LiskHQ/lisk-hub/issues/1805
🐛 Follower -> Bookmark
https://github.com/LiskHQ/lisk-hub/issues/1805
🐛 Account name text field got the green validation mark inside

**Comment by slaweet** [comment]
Good feedback, @Efefefef.
1), 4) - let's align the terminology and use "Account" in these places instead of "Wallet"
Screen Shot 2019-03-22 at 09 16 58 1

2) Date is ok for last block
3) Should redirect to "My wallet"
5) updated on https://github.com/LiskHQ/lisk-hub/issues/1780

**Pull request merged**